### PR TITLE
DBZ-3248 add log4j.properties file

### PR DESCRIPTION
**Pull request by fuxiao224**
Add log4j.properties file to debezium cassandra connector

### CODE REVIEW

File: src/main/resources/log4j.properties
Comment (bingqinzhou) [inline src/main/resources/log4j.properties:13]: Can we change `db2` here to be `cassandra` please?

File: src/main/resources/log4j.properties
Comment (bingqinzhou) [inline src/main/resources/log4j.properties:9]: My bad, please change the logging level here back to `INFO`.

File: src/main/resources/log4j.properties
Comment (fuxiao224) [reply src/main/resources/log4j.properties:13]: Sure! Will do!

**Comment by fuxiao224** [comment]
> One tiny comment. @bingqinzhou, could you merge it once that's addressed? Thx!
Thanks for your comment! I just resubmitted. FYI: It seems like Bingqin is having some difficulties logging into github recently. I'll followup with her tmr to see if she is able to log in now.

**Review by gunnarmorling — APPROVED** [review]
LGTM, thx. Could you only rework the commit using an email address linked to your GH account (so it's properly attributed to you) and then force-push. There shouldn't be the grey octocat next to the commit message if everything is alright. Thanks!

**Comment by fuxiao224** [comment]
@gunnarmorling Thanks for pointing that out! I didn't even realize this before. FYI @bingqinzhou has some issues logging into github recently, would you please merge it, or I'll merge it once you grant me committer access. Thanks!

**Comment by gunnarmorling** [comment]
Hey @fuxiao224, yepp, no problem, I'll merge it. But I'm still seeing the three commits with the Octocat icon. See here for a description: https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/why-are-my-contributions-not-showing-up-on-my-profile#your-local-git-commit-email-isnt-connected-to-your-account. Could you configure your local git so it uses an e-mail linked as described there, then rebase and squash all the commits into one and force-push to this PR? Thx!

**Comment by fuxiao224** [comment]
Thanks for hints! It took me a while to figure out, but finally was able to overwrite default configs and resubmit! Please take a look again @gunnarmorling

**Review by gunnarmorling — APPROVED** [review]

**Comment by gunnarmorling** [comment]
Applied, thx, @fuxiao224!

**Pull request merged**
